book/sources/exex/remote/build.rs

```diff
@@ -0,0 +1,4 @@
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    tonic_build::compile_protos("proto/exex.proto")?;
```

Comment:
for this to work in the CI, we also need to install `protoc`. I think it's not a big deal as it's quite lightweight, wdyt @mattsse ?

Reply:
refactoring the job and adding protoc in https://github.com/paradigmxyz/reth/pull/11618

Reply:
why does this need to run on ci?
yeah I guess if we want to do have well maintained bindings then we're kinda forced to do this. sounds reasonable